packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1077,12 +1077,15 @@ function hideOrUnhideAllChildren(finishedWork, isHidden) {
       if (node.tag === HostComponent) {
         if (hostSubtreeRoot === null) {
           hostSubtreeRoot = node;
-
-          const instance = node.stateNode;
-          if (isHidden) {
-            hideInstance(instance);
-          } else {
-            unhideInstance(node.stateNode, node.memoizedProps);
+          try {
+            const instance = node.stateNode;
+            if (isHidden) {
+              hideInstance(instance);
```

Comment:
Should we also do this for `(un)hideTextInstance` below?

Reply:
Yeah I missed that one